packages/services/src/models/notification-scanners/DueTasks.js

```diff
@@ -0,0 +1,79 @@
+import NotificationScannerBaseModel from './NotificationScannerBaseModel'
+import Network from '@aragonone/court-backend-server/build/web3/Network'
+import dateFormat from 'dateformat'
+
+class DueTasks extends NotificationScannerBaseModel {
+  async scan() {
+    let notifications = []
+    const oneDayBeforeNow = Math.floor((Date.now()-this._DAYS)/1000)
+    const threeDaysBeforeNow = Math.floor((Date.now()-(3*this._DAYS))/1000)
+    const query = `
+    {
+      committingRounds: adjudicationRounds(where: {state: Committing, createdAt_lt: ${oneDayBeforeNow}}, orderBy: createdAt) {
+        createdAt
+        dispute {
+          id
+        }
+        jurors (where: {commitment: null}) {
+          juror {id}
+        } 
+      }
+     	revealingRounds: adjudicationRounds(where: {stateInt_in: [1,2], createdAt_lt: ${threeDaysBeforeNow}}, orderBy: createdAt) {
+        createdAt
+        dispute {
+          id
+        }
+        jurors (where: {commitment_not: null, revealDate: null}) {
+          juror {id}
+        } 
+      }
+    }
```

Comment:
Would it be better to have two scanners for this? One for due commits and another for reveals?

Reply:
I was thinking one is better because we can potentially have fewer total emails

Reply:
Are we mixing due commits and due reveals in the same email for same jurors then?

Reply:
Yes, it was like this in the original email template actually:
![image](https://user-images.githubusercontent.com/26359933/82957629-c289e200-9fb3-11ea-9ad8-036541035bd4.png)